.github/workflows/application-health-checker.yml

```diff
@@ -34,17 +34,29 @@ jobs:
 
-      - name: Replace institution.super_admin_user_one.username in configuration.properties # Step to replace the placeholder institution_one.admin_user_one.username in configuration.properties
-        run: sed 's|'{INSTITUTION_SUPER_ADMIN_USER_ONE_USERNAME}'|'${{ secrets.AYS_INSTITUTION_SUPER_ADMIN_USER_ONE_USERNAME }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
+      - name: Replace institution_one.admin_user_two.email_address in configuration.properties # Step to replace the placeholder institution_one.admin_user_one.username in configuration.properties
+        run: sed 's|'{INSTITUTION_ONE_ADMIN_USER_TWO_EMAIL_ADDRESS}'|'${{ secrets.AYS_INSTITUTION_ONE_ADMIN_USER_TWO_EMAIL_ADDRESS }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
 
-      - name: Replace institution.super_admin_user_one.password in configuration.properties # Step to replace the placeholder institution_one.admin_user_one.password in configuration.properties
-        run: sed 's|'{INSTITUTION_SUPER_ADMIN_USER_ONE_PASSWORD}'|'${{ secrets.AYS_INSTITUTION_SUPER_ADMIN_USER_ONE_PASSWORD }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
+      - name: Replace institution_one.admin_user_two.password in configuration.properties # Step to replace the placeholder institution_one.admin_user_one.password in configuration.properties
+        run: sed 's|'{INSTITUTION_ONE_ADMIN_USER_TWO_PASSWORD}'|'${{ secrets.AYS_INSTITUTION_ONE_ADMIN_USER_TWO_PASSWORD }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
+
+      - name: Replace institution.test_admin.email_address in configuration.properties # Step to replace the placeholder institution.test_admin.email_address in configuration.properties
+        run: sed 's|'{INSTITUTION_TEST_ADMIN_EMAIL_ADDRESS}'|'${{ secrets.AYS_INSTITUTION_TEST_ADMIN_EMAIL_ADDRESS }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
+
+      - name: Replace institution.test_admin.password in configuration.properties # Step to replace the placeholder institution.test_admin.password in configuration.properties
+        run: sed 's|'{INSTITUTION_TEST_ADMIN_PASSWORD}'|'${{ secrets.AYS_INSTITUTION_TEST_ADMIN_PASSWORD }}'|g' configuration.properties >> temp_configuration.properties ; rm configuration.properties ; mv temp_configuration.properties configuration.properties
```

Comment:
@cagla88 parametrelerle ilgili BE için konuştuktan sonra buraya yansıtsak daha sağlıklı olabilir. Orada bir kaç düzenleme yapmamız gerekiyor

Reply:
@agitrubard github action environmentlara da eklemiştim tamamen unutmuşum değişiklik olacağını. BE taraftan sonra yeniden düzenlerim drafta alıyorum pr yi